client/blocks/comments/comment-actions.jsx

```diff
@@ -46,6 +49,15 @@ const CommentActions = ( {
 
 	return (
 		<div className="comments__comment-actions">
+			{ showReadMore &&
+				<button className="comments__comment-actions-read-more" onClick={ noop }>
```

Comment:
you may be interested in [expandComments](https://github.com/Automattic/wp-calypso/blob/master/client/state/comments/actions.js#L219) for the `onClick` handler